packages/store/src/ivy/ivy-enabled-in-dev-mode.ts

```diff
@@ -22,8 +23,8 @@ export function setIvyEnabledInDevMode(): void {
     // `try-catch` will also handle server-side rendering, as
     // `window is not defined` will not be thrown.
     const ng = (window as any).ng;
-    const _ivyEnabledInDevMode =
-      !!ng && typeof ng.getComponent === 'function' && typeof ng.markDirty === 'function';
+    const _viewEngineEnabled = (!!ng && !!ng.probe) || !!ng.coreTokens;
```

Comment:
```suggestion
    const _viewEngineEnabled = !!ng.probe && !!ng.coreTokens;
```

Sorry, last change :smile: